src/plugins/data_source/server/audit/logging_auditor.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AuditableEvent, Auditor, Logger, OpenSearchDashboardsRequest } from 'src/core/server';
+
+export class LoggingAuditor implements Auditor {
+  constructor(
+    private readonly request: OpenSearchDashboardsRequest,
+    private readonly logger: Logger
+  ) {}
+
+  public withAuditScope(name: string) {}
+
+  public add(event: AuditableEvent) {
```

Comment:
I think we need more than type and message in the log line. For example, username or credential id?

Reply:
for now let's leave that. need security plugin's support to fully enable the log for authentication related stuff.